.github/workflows/go.yml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2022 Henry Bubert
```

Comment:
Could you change the copyright to yourself? :)

Reply:
Done :upside_down_face: